.claude/commands/jira-register.md

```
@@ -0,0 +1,232 @@
+## 사용법
+
+```bash
+/jira-register {PARENT-ISSUE-KEY}
+/jira-register MEMBER-123
+```
+
+---
+
+## 실행 프로세스
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)